demo/index.html

```diff
@@ -22,7 +24,49 @@
       display: block;
     }
   </style>
-  <link href="https://fonts.googleapis.com/css?family=Montserrat:300,400,700|Roboto:300,400,700" rel="stylesheet">
+  <!-- <link href="https://fonts.googleapis.com/css?family=Montserrat:300,400,700|Roboto:300,400,700" rel="stylesheet"> -->
```

Comment:
To extract a configuration parameter to add additional header in order to add the custom favicon or fonts. Monserrat and Roboto by default.